### DISCUSSION TITLE
Spectators are misaligned in Tab menu

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14939] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

cant upload picture here due to network issue

### Reproduction steps

.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
![20241024222837](https://github.com/user-attachments/assets/45dc66de-5583-410b-8817-624d7f8cd20b)

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly. Fix coming in the next update. Closing discussion.
